www/js/general.js

```diff
@@ -91,6 +91,18 @@ shinyjs.init = function() {
         });
     });
 
+    $(document).ready(function(){
+        $('#ADD_SIZE2').click(function(){
+            console.log('testing');
```

Comment:
one of these blocks is superfluous (the other starts on line 703). i'll remove the extra one. also note that any `console.log` calls used for testing can be removed before a push